packages/headless/src/controllers/smart-snippet/headless-smart-snippet.ts

```diff
@@ -80,16 +94,20 @@ export function buildSmartSnippet(engine: Engine<object>): SmartSnippet {
     },
 
     expand() {
-      // TODO
+      engine.dispatch(logExpandSmartSnippet());
+      // TODO manage state expanded
     },
     collapse() {
-      // TODO
+      engine.dispatch(logCollapseSmartSnippet());
+      // TODO manage state expanded
     },
     thumbsUp() {
-      // TODO
+      engine.dispatch(logLikeSmartSnippet());
```

Comment:
I think like/dislike are great names for methods, and they fit the state + analytics

Reply:
I cd'ed into headless packages, npm i coveo.analytics, cd'ed back to root, and ran `npm run setup`, `npm run build` 🤷